linux/powershell/PSCloudShellUtility/tips.json

```diff
@@ -1,5 +1,6 @@
 {
   "items": [
-    "Azure Cloud Shell now includes Predictive IntelliSense! Learn more: https://aka.ms/CloudShell/IntelliSense"
+    "Azure Cloud Shell now includes Predictive IntelliSense! Learn more: https://aka.ms/CloudShell/IntelliSense",
+    "SqlServer has been updated to its latest version!"
```

Comment:
Should we be more specific and say that the Powershell module for SQLServer has been updated to version X?

Reply:
@dsajanice - you could say "v22" i.e. just mention the major jump.